addons/binding/org.openhab.binding.pioneeravr/OSGI-INF/PioneerAvrDiscovery.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.binding.pioneeravr.discovery">
+   <implementation class="org.openhab.binding.pioneeravr.discovery.PioneerAvrDiscoveryParticipant"/>
+   <service>
+      <provide interface="org.eclipse.smarthome.config.discovery.UpnpDiscoveryParticipant"/>
+      <provide interface="org.osgi.service.cm.ManagedService"/>
+   </service>
+   <property name="service.pid" type="String" value="org.openhab.binding.pioneeravr"/>
```

Comment:
the service pids should rather be org.openhab.xxx - but here I would suggest to remove it completely and adapt the component name instead.